src/css/_docs.scss

```diff
@@ -435,6 +435,7 @@
     font-size: 15px;
     line-height: 17px;
     position: relative;
+    border-top: 1px solid #333;
```

Comment:
Well, the background here is #000, so the #333 is a soft grey line for inactive chapters